src/main/kotlin/no/nav/syfo/domain/SenOppfolgingKandidat.kt

```diff
@@ -19,25 +21,30 @@ data class SenOppfolgingKandidat private constructor(
         createdAt = OffsetDateTime.now(),
         varselAt = varselAt,
         svar = null,
+        publishedAt = null,
     )
 
     fun addSvar(svar: SenOppfolgingSvar): SenOppfolgingKandidat = this.copy(
         svar = svar,
     )
 
+    fun publish(): SenOppfolgingKandidat = this.copy(publishedAt = nowUTC())
```

Comment:
Gir det mening å ha `publishedAt` i domeneobjektet? Har tenkt på dette som en teknisk detalj som er knyttet opp mot hvordan vi velger å eksponere dette for omverdenen🤔

Reply:
Enig i det :+1: